### PR TITLE
fix(mls): disable mls everywhere and only considering the BE flags (AR-2653)

### PR DESCRIPTION
**Pull request by mchenani**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2653" title="AR-2653" target="_blank">BugAR-2653</a>  Hide MLS options in versions that use V1
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?
We were overriding the MLS enabled values for different envs to make it easy for test.
The QA asked to disable all of them and only rely on the API Version and the feature config on the team settings.

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: default.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "safe_logging": false,
         "private_build": true,
         "development_api_enabled": true,
-        "mls_support_enabled": true,
+        "mls_support_enabled": false,
```
Comment (typfel) [inline default.json]: This disables MLS completely we won't even look at the feature config.

**Review by typfel — CHANGES_REQUESTED** [review]
Did I understand correctly that QA want the UI MLS to be hidden/visible based on the MLS feature config?

Then we should leave the MLS build config enabled and instead rely on the `IsMLSEnabledUseCase`.

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1333/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5498125)(5498125)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5498125)(5498125)-dev-compat.apk)

File: default.json (hunk 1 above)
Comment (mchenani) [reply default.json]: Yes, correct. Updated it correctly.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1333/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5498391)(5498391)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5498391)(5498391)-dev-compat.apk)

**Review by typfel — APPROVED** [review]

**Pull request merged**